packages/website/docs/using-query-suggestions-plugin.md

```diff
@@ -2,7 +2,137 @@
+
+For now, `plugins` is an empty array, but you'll learn how to add the Query Suggestions plugin next.
+
+## Adding Query Suggestions
+
+The [`autocomplete-plugin-query-suggestions`](createQuerySuggestionsPlugin) package provides the [`createQuerySuggestionsPlugin`](createQuerySuggestionsPlugin) function for creating a Query Suggestions plugin out-of-the-box.
+
+It requires an [Algolia search client](https://www.algolia.com/doc/api-client/getting-started/install/javascript/) initialized with an [Algolia application ID and API key](https://www.algolia.com/doc/guides/sending-and-managing-data/send-and-update-your-data/how-to/importing-with-the-api/#application-id) and an `indexName`. The `indexName` is the name of your [Query Suggestions](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/query-suggestions/js/) index. **You must have a populated [Query Suggestions](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/query-suggestions/js/#implementing-query-suggestions) index to use this plugin.**
+
+```diff
+import { autocomplete } from '@algolia/autocomplete-js';
++ import { createQuerySuggestionsPlugin } from '@algolia/autocomplete-plugin-query-suggestions';
++
++ const searchClient = algoliasearch(
++   'yourAppID',
++   'yourSearchApiKey'
++ );
++
++ const querySuggestionsPlugin = createQuerySuggestionsPlugin({
++   searchClient,
++   indexName: 'yourQuerySuggestionsIndexName',
++   },
++ });
++
+autocomplete({
+  container: '#autocomplete',
+-  plugins: [],
++  plugins: [querySuggestionsPlugin],
+  openOnFocus: true,
+});
+```
+
+You can optionally pass a `getSearchParams` function to apply [Algolia query parameters](https://www.algolia.com/doc/api-reference/api-parameters/) to the suggestions returned from the plugin. This is particularly useful if you need to align your Query Suggestions with other sections displayed in the autocomplete, like [recent searches](using-recent-searches-plugin).
+
+```diff
+import { autocomplete } from '@algolia/autocomplete-js';
+import { createQuerySuggestionsPlugin } from '@algolia/autocomplete-plugin-query-suggestions';
+
+const searchClient = algoliasearch(
+  'yourAppID',
+  'yourSearchApiKey'
+);
+
+const querySuggestionsPlugin = createQuerySuggestionsPlugin({
+  searchClient,
+  indexName: 'yourQuerySuggestionsIndexName',
++ getSearchParams() {
++   return recentSearchesPlugin.data.getAlgoliaSearchParams({
++     hitsPerPage: 5,
++   });
+  },
+});
+
+autocomplete({
+  container: '#autocomplete',
+  plugins: [querySuggestionsPlugin],
+  openOnFocus: true,
+});
+```
+This creates a basic Query Suggestions implementation. Try it out below:
+
+<AutocompleteExample
+  plugins={[querySuggestionsPlugin]}
+  openOnFocus={true}
+/>
+
+:::note
+
+These suggestions are based on a [public dataset of BestBuy products](https://github.com/algolia/datasets/tree/master/ecommerce).
+
+:::
+
+## Next steps
```

Comment:
We don't display code snippets of a RS + QS integration anywhere yet, and I believe a section about it would make sense here. Wdyt?

Reply:
Good idea! Is there already an example I can refer to, or should I work on.

Reply:
In our main example we use RS + QS: https://github.com/algolia/autocomplete.js/blob/56cbb20c7f18e1c14404aaf9fccec6fdd1811d5b/examples/js/app.tsx

Reply:
Sorry, I totally misunderstood what you were referring to.

I'm not sure we need to display these here since there's a separate RS guide and a guide on showing RS/QS/Predefined together. (The bottom of this page references it "To learn how to display multiple sections in one autocomplete, read the guide on creating multi-source autocompletes.")

To keep things modular, I think the RS And QS pages should be standalone. WDYT?